### PR TITLE
add index.android.js file

### PR DESCRIPTION
**Pull request by mng12689**

### CODE REVIEW

**Comment by mng12689** [comment]
this allows us to simply call isSupported before making subsequent calls to other functions in this lib and not have to worry about what platform were running on

File: index.android.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  /**
+   * An initial action will be available if the app was cold-launched
+   * from an action.
+   *
+   * The first caller of `popInitialAction` will get the initial
+   * action object, or `null`. Subsequent invocations will return null.
+   */
+  popInitialAction: function () {
+    console.warn('react-native-quick-actions popInitialAction not supported on iOS');
```
Comment (jordanbyron) [inline index.android.js]: @mng12689 I think these are supposed to say Android, not iOS:

> react-native-quick-actions popInitialAction is not supported on Android

File: index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+var Platform = require('react-native').NativeModules;
+var entryFile = Platform.OS === 'ios' ? require('./index.ios.js') : require('./index.android.js');
+module.exports = entryFile;
```
Comment (jordanbyron) [inline index.js]: This is a little confusing and if I am reading it correctly I think it loads `index.android.js` regardless of platform. Perhaps it should be re-written:

```js
const React = require('react-native');
const { Platform } = React;

if (Platform.OS === 'android') {
  require('./index.android.js');
} else if (Platform.OS === 'ios') {
  require('./index.ios.js');
}
```

**Comment by jordanbyron** [comment]
Closed due it inactivity.

**Pull request closed without merging**